vj4/handler/fs.py

```diff
@@ -25,10 +25,15 @@ class FsGetHandler(base.Handler):
     self.response.content_type = grid_out.content_type or 'application/octet-stream'
     # FIXME(iceboy): For some reason setting response.content_length doesn't work in aiohttp 2.0.6.
     self.response.headers['Content-Length'] = str(grid_out.length)
+    ext = mimetypes.guess_extension(self.response.content_type)
+    if not ext:
+      ext = ''
+    self.response.headers.add('Content-Disposition',
```

Comment:
Can we reuse handler.binary?

Reply:
No, because this contains cache-control, block write, ...

Reply:
ok